q2_feature_table/tests/filter/test_filter_features.py

```diff
@@ -45,12 +45,12 @@ def test_min_frequency(self):
                          ['S1', 'S2', 'S3'])
         self.assertEqual(actual, expected)
 
-        # filter all
+        # filter all raising ValueError
         table = Table(np.array([[0, 1, 1], [1, 1, 2]]),
                       ['O1', 'O2'],
                       ['S1', 'S2', 'S3'])
-        actual = filter_features(table, min_frequency=5)
-        self.assertTrue(actual.is_empty())
+        with self.assertRaisesRegex(ValueError, 'empty'):
```

Comment:
This is good! I would include a bit more text for the regex you're expecting to see within this value error though, just to make sure there's no chance of overlap with another existing value error. Same suggestion for all of your test cases that include this assertion!